### DISCUSSION TITLE
anybody having issues when setting split_direction = "horizontal"?

### DISCUSSION THREAD

**Discussion by jmlucjav** [mistweaverco/kulala.nvim#323] [Q&A] [answered]
hi,

I don't know if this is a bug, cause I cannot reliably reproduce it always...
What I see is:

- if I dont specify split_direction, it creates thorizontal split fine in my terminal nvim, and also neovide
- if I specify 	split_direction = "horizontal"...neovide sometimes creates the split fine (not always), and terminal never does it. It opens a buffer but not in split.

Anyone has some ideas??
Thanks for the great project!

**Comment by YaroSpace** [comment, marked as answer]
Our config currently has only options for `display_mode = split|float`. It you specify anything else, it will open a split.

**Reply by YaroSpace** [reply]
Sorry, we do actually have option `split_direction = "horizontal|vertical"` and it should be working fine.

**Reply by jmlucjav** [reply]
Yeah I didn't understand your previous reply. Anyway, I cannot consistently reproduce this,so feel free to close
